INVESTIGATION_SUMMARY.md

```
@@ -0,0 +1,225 @@
+# From inside container - WORKS FINE  
+$ docker exec qwen curl http://localhost:8890/health
+{"status": "healthy", "model_loaded": true}
+
+# Container logs show successful internal requests
+INFO: 127.0.0.1:34406 - "GET /health HTTP/1.1" 200 OK
+INFO: 172.24.0.1:36260 - "POST /tts HTTP/1.1" 200 OK
+```
+
+**Why This Causes Missing Audio:**
+
+1. Audify makes health check request to Qwen API
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)